modules/installing-oci-about-agent-based-installer.adoc

```diff
@@ -6,12 +6,14 @@
 [id="installing-oci-about-agent-based-installer_{context}"]
 = The Agent-based Installer and {oci} overview
 
-You can install an {product-title} cluster on {oci-first} by using the Agent-based Installer. Both Red Hat and Oracle test, validate, and support running {oci} and {ocvs-first} workloads in an {product-title} cluster on {oci}.
+You can install an {product-title} cluster on {oci-first} by using the Agent-based Installer. Both Red{nbsp}Hat and Oracle test, validate, and support running {oci} workloads in an {product-title} cluster.
 
-The Agent-based installer provides the ease of use of the Assisted Installation service, but with the capability to install a cluster in either a connected or disconnected environment.
+The Agent-based Installer provides the ease of use of the Assisted Installation service, but with the capability to install a cluster in either a connected or disconnected environment.
 
 The following diagrams show workflows for connected and disconnected environments:
 
+// Based on the actual workflow throughout this assembly and Oracle docs, some of the things in these images seem to be out of order. These images can probably be updated later.
```

Comment:
Assuming this is ok for now? might want to rephrase the note as something like "todo: verify the order of steps in these images"

Reply:
I think these probably need be updated, so a TODO would be appropriate here